plugins/tableselection/plugin.js

```diff
@@ -379,22 +379,18 @@
 	}
 
 	function fakeSelectionDragHandler( evt ) {
-		var table = evt.data.getTarget().getAscendant( 'table', true );
+		var table = evt.data.getTarget().getAscendant( 'table', true ),
+			editor = evt.editor || evt.sender.editor;
 
-		// (#2945)
-		if ( table && table.hasAttribute( ignoredTableAttribute ) ) {
+		// (#547)
+		if ( table && evt.name == 'dragstart' ) {
+			table.data( ignoredTableAttribute, '' );
```

Comment:
The `element.data()` method adds `data-` prefix so you will end up with `data-data-cke-tableselection-ignored`. This means this code doesn't add proper ignore attribute and still works which means the attribute may be not needed 🤔 

Also the condition for `#2945` was removed so you need special attention here to se if nothing gets broken. Btw. I think it will still work with this condition present.

Reply:
I would consider the other way around approach too (not saying it will be better but worth checking) - instead of just always allowing dragging (this is what happens after this change basically), maybe don't prevent `dragStart` event in more case. The same it is done for `table && table.hasAttribute( ignoredTableAttribute )` or `!cell || cell.hasClass( fakeSelectedClass )` - it should be also possible to be done for image or text elements.

Reply:
Firstly, you're obviously right - the attribute was misused and useless 👎 But later I found out that trying to reuse it in this case isn't trival, as after drop we want to remove the temporal attribute, and I didn't see a way to distinguish whether it was added by user with #2945 in mind, or just added for dragging, so I decided to restore removed conditions and mimic that solution but with a new, dedicated attribute (I came up with `cke-table-dragging-active`), except for the part where I control turning it on and off.
As for the second comment, during tests I didn't encounter and can't really think of any edge-case scenario I would like to cut out, so if you have any of them on mind or find it during review please let me know. For now I would like to give a go to this solution.